.gitignore

```diff
@@ -1,3 +1,5 @@
 /files/*
 !.keep
 /log
+
+.idea
```

Comment:
@ninewise heeft daar _meningen_ over

Reply:
Je wil die in je globale `~/.gitignore` steken uiteraard.
```suggestion
```